features/extension-mgt/org.wso2.carbon.identity.extension.mgt.feature/resources/extensions/connections/custom-authentication/info.json

```diff
@@ -0,0 +1,10 @@
+{
+  "id": "custom-authentication",
+  "name": "Custom Authentication",
+  "description": "Enable login for users with external authentication service.",
+  "image": "",
+  "category": "DEFAULT",
+  "displayOrder": -1,
+  "tags": [],
```

Comment:
Add custom tag

Reply:
This is updated with 7d2336487ac11a54b528c7c248af61bb678b98e8